### PR TITLE
Adding placeholder

### PR DESCRIPTION
**Pull request by aliazizi**
Adding placeholder to CNRichTextEditor props

### CODE REVIEW

**Comment by aliazizi** [comment]
Also I add new props call `textInputStyle` with this prop we can change style for textinput and placeholder.
So I can now change placeholder fontFamily and fontSize

File: src/CNRichTextEditor.js [hunk 1]
```diff
@@ -424,9 +424,10 @@ class CNRichTextEditor extends Component {
             returnKeyType={this.props.returnKeyType}
             foreColor={this.props.foreColor}
             styleList={styles}
-            style={{
+			placeholder={index === 0 && this.props.placeholder}
```
Comment (imnapo) [inline src/CNRichTextEditor.js]: I'm using expo on IOS and the package crashes for situations like uploading image. I think its because of this line.

Comment (imnapo) [inline src/CNRichTextEditor.js]:
```suggestion
			placeholder={index === 0 ? this.props.placeholder : undefined}
```

Comment (imnapo) [reply src/CNRichTextEditor.js]: this solved my issue

**Comment by aliazizi** [comment]
Thanks for fixed, I updated my pull request

**Pull request merged**